fmriprep/cli/sample_openfmri_tasks_list.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+A tool to generate a tasks_list.sh file for running fmriprep
+on subjects downloaded with datalad with sample_openfmri.py
+
+
+"""
+
+import os
+import glob
+
+CMDLINE = """\
+{fmriprep_cmd} {bids_dir}/{dataset_dir} {dataset_dir}/out/ participant \
```

Comment:
We should have an absolute path for the base output directory, instead of using the CWD.

```Python
{output_dir}/{dataset_dir}
```

If you want to have `/out` (I prefer `/derivatives`) after that, that's fine. But I think `output_dir` should be specified by command line option. (The default could be `os.getcwd()` if you like.)

Reply:
This was already in one version I have on ls5. We are in sync.